src/constants/eip712.ts

```diff
@@ -2,11 +2,11 @@
  * EIP712 domain name.
  * @see {@link https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator EIP712 doc}
  */
-export const DOMAIN_NAME = "LooksRareProtocol";
+export const DOMAIN_NAME = "Hypercerts";
```

Comment:
this was changed last time as well, and it broke signatures as the API needs to be updated too, so I rolled it back. if we want to change it, it needs to be rolled out at the same time.

Reply:
Okay, fair! I'll leave this to you to coordinate :)